AnimalPicker.java

```diff
@@ -18,13 +18,15 @@ public static void main(String[] args) {
 }*/
  if(userInput1.equalsIgnoreCase("cat")) {
 	
-	System.out.println("            _       \n   ___ __ _| |_ ___ \n  / __/ _` | __/ __|\n | (_| (_| | |_\\__ \\\n  \\___\\__,_|\\__|___/");
-	
-}else if(userInput1.equalsIgnoreCase("fish")) {
+	//System.out.println("            _       \n   ___ __ _| |_ ___ \n  / __/ _` | __/ __|\n | (_| (_| | |_\\__ \\\n  \\___\\__,_|\\__|___/");
+	System.out.println("CAT ASCII ART HERE");
```

Comment:
I dont think you need:  String userInput = "";  on line 7 because you have userInput1 scanner.